cmd/candidsrv/config.yaml

```diff
@@ -11,10 +11,14 @@ max-mgo-sessions: 300
 request-timeout: 2s
 private-addr: localhost
 admin-agent-public-key: iKp24EL2Aj9MQfRkpwzp7rz7Zf3QZsEzWGpoWT3OK2w=
+rp-display-name: Candid
```

Comment:
can we name these mfa-rp-display-name etc. The problem with RP is that the term occurs in pretty much any authentication protocol so we will probably want another RP in a different context in future.

Reply:
done